packages/manager/modules/order/src/cart/order-cart.service.js

```diff
@@ -93,19 +93,19 @@ export default class OrderCartService {
    *
    * @param  {string} cartId          Identification number of the cart
    * @param  {string} productName     Name of the product
-   * @param  {string} productPlanCode Plan code of the selected product (ex:
+   * @param  {string} planCode        Plan code of the selected product (ex:
    * for the product 'webHosting', get options for the plan code 'cloudweb1')
    * @return {Promise<Array>}         Promise of the product available list of
    * options
    */
-  getProductOptions(cartId, productName, productPlanCode) {
+  getProductOptions(cartId, productName, planCode) {
     return this.OvhApiOrder.Cart()
       .Product()
       .v6()
       .getOptions({
         cartId,
         productName,
-        ...productPlanCode,
+        planCode,
```

Comment:
@ganeshkumar1989, we don't need to destruct it like for `productPlanCode` ?

Reply:
This method is being called with only the plan code and it does not need to be an object. Hence, we are now calling this method with the plancode.